src/frontend/MyRoutes/ProtectedRoute.js

```diff
@@ -2,7 +2,7 @@ import { Navigate } from "react-router-dom";
 import { useAuth } from "../index";
 function ProtectedRoute({ children }) {
   const { auth } = useAuth();
-  return auth.status ? children : <Navigate to="/signin" replace />;
+  return auth.status ? children : <Navigate to="/join" replace />;
```

Comment:
Suggestion : You can use dynamic routing as redirecting to previous url by using `state` and `location`

Reply:
yup on it. Thank you